### ISSUE TITLE
[Paper] Would I Lie To You? Inference Time Alignment of Language Models using
  Direct Preference Heads

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed-dev#104]
{
  "arxivId": "2405.20053",
  "url": "https://arxiv.org/abs/2405.20053",
  "title": "Would I Lie To You? Inference Time Alignment of Language Models using\n  Direct Preference Heads",
  "authors": "Avelina Asada Hadji-Kyriacou, Ognjen Arandjelovic",
  "abstract": "Pre-trained Language Models (LMs) exhibit strong zero-shot and in-context\nlearning capabilities; however, their behaviors are often difficult to control.\nBy utilizing Reinforcement Learning from Human Feedback (RLHF), it is possible\nto fine-tune unsupervised LMs to follow instructions and produce outputs that\nreflect human preferences. Despite its benefits, RLHF has been shown to\npotentially harm a language model's reasoning capabilities and introduce\nartifacts such as hallucinations where the model may fabricate facts. To\naddress this issue we introduce Direct Preference Heads (DPH), a fine-tuning\nframework that enables LMs to learn human preference signals through an\nauxiliary reward head without directly affecting the output distribution of the\nlanguage modeling head. We perform a theoretical analysis of our objective\nfunction and find strong ties to Conservative Direct Preference Optimization\n(cDPO). Finally we evaluate our models on GLUE, RACE, and the GPT4All\nevaluation suite and demonstrate that our method produces models which achieve\nhigher scores than those fine-tuned with Supervised Fine-Tuning (SFT) or Direct\nPreference Optimization (DPO) alone.",
  "timestamp": "2024-12-15T17:25:17.781Z",
  "rating": "novote"
}

**Comment by dmarx** [comment]
✅ Paper processed and added to the database. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.

**Comment by dmarx** [comment]
✅ Event processed and recorded. Closing this issue.
